types/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ All notable changes to this project will be documented in this file.  The format
 
 ## [Unreleased]
 
+### Changed
+* Extended `UnbondingPurses` to take a new field `new_validator` which represents the validator to whom tokens will be re-delegated.
```

Comment:
You forgot to mention that we also changed:
* RPC API
* SSE data
* clients (in this PR only Rust client but the change forces the update to all the other SDKs)